k8s/demo/cassandra/README.md

```diff
@@ -0,0 +1,223 @@
+pvc-e7d18817-c6bb-11e7-a0eb-000c298ff5fc-rep-3006965094-cntx5    1/1       Running   0          4h
+pvc-e7d18817-c6bb-11e7-a0eb-000c298ff5fc-rep-3006965094-mhsjt    1/1       Running   0          4h
+```
+
+```
+test@Master:~/openebs/k8s/demo/cassandra$ kubectl get svc
+NAME                                                CLUSTER-IP       EXTERNAL-IP   PORT(S)             AGE
+cassandra                                           None             <none>        9042/TCP            5h
+kubernetes                                          10.96.0.1        <none>        443/TCP             14d
+maya-apiserver-service                              10.102.92.217    <none>        5656/TCP            14d
+pvc-1c16536c-c6bc-11e7-a0eb-000c298ff5fc-ctrl-svc   10.107.177.156   <none>        3260/TCP,9501/TCP   4h
+pvc-d1fee1af-bf73-11e7-9be8-000c298ff5fc-ctrl-svc   10.98.87.196     <none>        3260/TCP,9501/TCP   9d
+pvc-dfe98642-bf73-11e7-9be8-000c298ff5fc-ctrl-svc   10.101.17.255    <none>        3260/TCP,9501/TCP   9d
+pvc-e7d18817-c6bb-11e7-a0eb-000c298ff5fc-ctrl-svc   10.108.47.234    <none>        3260/TCP,9501/TCP   4h
+```
+
+Note: It may take some time for pods to start as the images need to be pulled and instantiated. This is also
+dependent on the network speed.
+
+## Verify successful Cassandra deployment
+
+The verification procedure can be carried out in a series of steps, starting from listing the functional 
+replicas, followed by creation and deletion of test data in the cassandra database.
+
+### Step-1: Install the Cqlsh utility
+
+Cqlsh is a Python based utility that enables you to execute CQL. The CQL (Cassandra Query Language) is a 
+declarative language that enables users to query Cassandra using a semantics similar to SQL. 
+
+Install the python-minimal and python-pip apt packages (if not available) and perform a pip installtion of 
+Csqlsh.
+
+```
+sudo apt-get install -y python-minimal python-pip 
+pip install cqlsh
+```
+
+Note: Installation of Csqlsh may take a few minutes (typically, the cassandra-driver package takes time to 
+be downloaded and setup). 
+
+### Step-2: Verify replica status on Cassandra
+
+```
+test@Master:~$ kubectl exec cassandra-0 -- nodetool status
+Datacenter: DC1-K8Demo
+======================
+Status=Up/Down
+|/ State=Normal/Leaving/Joining/Moving
+--  Address    Load       Tokens       Owns (effective)  Host ID                               Rack
+UN  10.36.0.6  103.83 KiB  32           100.0%           e013c19d-9c6f-49cd-838e-c69eb310f88e  Rack1-K8Demo
+UN  10.44.0.3  83.1 KiB    32           100.0%           1d2e3b79-4b0b-4bf9-b435-fcfa8be8a603  Rack1-K8Demo
+```
+
+A status of "UN" implies Up and Normal. The "Owns" column suggests the data distribution percentage for the
+content placed into the Cassandra keyspaces.In the current example,we have chosen a replica count of 2 due to 
+which the data is evenly distributed/copies maintained.
```

Comment:
due to which the data = so that the data
distributed/copies = distributed and copies

Reply:
"so that the data" - not the original intention. We were constrained by node resources because of which 2 replicas are chosen, and hence the data distribution pattern.

Updated second part (and instead of /)